### PR TITLE
Add GitHub workflow for test_missing_migrations

### LINKED ISSUE #1553: Travis CI -> GitHub workflow: test-for-missing-migrations
Kiwi TCMS needs help migrating certain test jobs from Travis CI into GitHub workflows. In particular the `test-for-missing-migrations` job.

**Definition of Done**:
- "test-for-missing-migrations" job has been removed from Travis CI
- "sanity / test-for-missing-migrations" job is enabled for GitHub workflows. See `.github/workflows/lint.yml` and create a `sanity.yml`
- The new job reports its result on the pull request

*This issue is part of Kiwi TCMS open source bounty program. For more information see the link(s) in [bounty-program milestone](https://github.com/kiwitcms/Kiwi/milestone/4)*

### PR DESCRIPTION
**Pull request by souravsingh**
Attempt to fix #1553

### CODE REVIEW

File: .github/workflows/sanity.yml [hunk 1]
```diff
@@ -24,3 +24,23 @@ jobs:
...
+        
+  test-for-missing-migrations:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
+      matrix:
+        python-version: [3.7]
+
+    steps:
+      - uses: actions/checkout@v3
```
Comment (RMadjev) [inline .github/workflows/sanity.yml:36]: Maybe its something wrong with version?

**Comment by RMadjev** [comment]
@souravsingh Please, apply the last change without an extra commit.

**Comment by souravsingh** [comment]
@RMadjev Do I need to make a new PR for this?

**Comment by RMadjev** [comment]
> @RMadjev Do I need to make a new PR for this?

No, you have to reset the last two commit you made, apply the version change to "Add workflow for missing migrations" and then force push the changes, this will automatically ref. current PR.

**Comment by atodorov** [comment]
> No, you have to reset the last two commits you made

@souravsingh this is called "to squash the commits", check this out: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by souravsingh** [comment]
Making another pull request.

**Pull request closed without merging**